src/nvim/fold.c

```diff
@@ -2167,7 +2167,7 @@ static linenr_T foldUpdateIEMSRecurse(garray_T *gap, int level,
         }
       }
       if (lvl < level + i) {
-        foldFind(&fp->fd_nested, flp->lnum - fp->fd_top, &fp2);
+        (void)foldFind(&fp->fd_nested, flp->lnum - fp->fd_top, &fp2);
```

Comment:
Even if `foldFind` fails (and indicates so by a return code), `fp` or `fp2` are set up correctly to the last entry of the 1st argument, so the return code can be ignored. Note that checking for `fp2 != NULL` is not strictly needed for this reason.